.github/workflows/tests.yml

```diff
@@ -36,10 +36,10 @@ jobs:
         run: build_scripts/run_shellcheck.sh
 
       - name: Run flake8
-        run: flake8
+        run: flake8 
 
-      - name: Run pytest (excluding integration tests)
-        run: pytest --verbose -m "not integration" --timer-top-n 10
+      - name: Run pytest Run All Other Tests (excluding integration and e2e)
+        run: pytest --verbose -m "not integration and not smoke and not e2e" --timer-top-n 10
```

Comment:
Is your smoke test just an instance of e2e tests? Maybe saying 'not smoke' and 'not e2e' is redundant? I'm not sure.